src/LuaSystemBody.cpp

```diff
@@ -276,6 +276,28 @@ static int l_sbody_attr_apoapsis(lua_State *l)
 	return 1;
 }
 
+
+/*
+ * Attribute: orbitPeriod
+ *
+ * The orbit of the body, around its parent, in days
```

Comment:
Eh, it's an API description. If the reader doesn't understand what "float" means they've got other problems...